server/src/main/java/org/apache/cassandra/sidecar/modules/AuthModule.java

```diff
@@ -157,6 +163,23 @@ VertxRoute chainAuthHandler(Vertx vertx,
                                                  .handler(chainAuthHandler));
     }
 
+    @DELETE
+    @Path(ApiEndpointsV1.INVALIDATE_CACHE_ROUTE)
+    @Operation(summary = "Invalidate authentication or authorization cache",
```

Comment:
We can use this endpoint for invalidating all caches, can you update summary to be generic?